osu.Game/Overlays/VolumeOverlay.cs

```diff
@@ -86,16 +87,10 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
-            volumeMeterMaster.Bindable.ValueChanged += _ => settingChanged();
-            volumeMeterEffect.Bindable.ValueChanged += _ => settingChanged();
-            volumeMeterMusic.Bindable.ValueChanged += _ => settingChanged();
-            muteButton.Current.ValueChanged += _ => settingChanged();
-        }
-
-        private void settingChanged()
-        {
-            Show();
-            schedulePopOut();
+            volumeMeterMaster.Bindable.ValueChanged += _ => Show();
+            volumeMeterEffect.Bindable.ValueChanged += _ => Show();
+            volumeMeterMusic.Bindable.ValueChanged += _ => Show();
+            muteButton.Current.ValueChanged += _ => Show();
```

Comment:
Maybe override `Show()` to `schedulePopOut()` instead of `PopIn()`. If it's already visible setting the `State` will return early and thus not trigger `PopIn()` again.